modules/ROOT/partials/depl-examples/ubuntu-compose/shared-setup.adoc

```diff
@@ -0,0 +1,442 @@
+
+User and User Access Management::
+The following embedded services are well suited for home use and smaller businesses though Infinite Scale can be configured to use external products which is relevant for bigger installations and not covered here.
++
+--
+* Infinite Scale has an embedded identity management (IDM footnote:[See the xref:{s-path}/idm.adoc[IDM, window=_blank] service for more details]) which takes care of creating, storing, and managing user identity information.
+
+* In addition, it also has an embedded identity provider (IDP footnote:[See the xref:{s-path}/idp.adoc[IDP, window=_blank] service for more details]) to track and manage user identities, as well as the permissions and access levels associated with those identities.
+--
+
+end::shared_1[]
+
+
+tag::hetzner_only_1[]
+
+== Prepare Hetzner
+
+=== SSH Key
+
+To be prepared accessing the configured server, we recommend using key based authentication for ssh instead using user and password. This is not only beneficial for security reasons but also because you can define the public key to be installed  during the initial server configuration.
+
+Follow the https://www.ssh.com/academy/ssh/keygen[ssh-keygen guide] to generate the required keys. We recommend, if possible, using the `ed25519` algorithm. The keys are now located in `~/.ssh`.
+
+When using Putty (Windows) to access your server, you must convert the private key generated into the `ppk` format to be usable for Putty. Read the  https://www.puttygen.com[puttygen] guide to do so.
+
+After the server has been created, you can copy new private keys to the server by adding them into `~/.ssh/authorized_keys` file.
+
+=== Login to Hetzer
+
+If you not already have an account on https://www.hetzner.com[Hetzner], register for free and log in.
+
+image::depl-examples/ubuntu-compose/hetzner-register-login.png[Hetzner Login or register, width=250]
+
+=== Configure and Order the Desired Server
+
+After logging in, select from the selector on the top right the `cloud` item.
+
+image::depl-examples/ubuntu-compose/hetzner-select-cloud.png[Select Cloud, width=150]
+
+Then, you either can use an existing project, if you have one, or create a `new project`.
+
+image::depl-examples/ubuntu-compose/hetzner-create-new-project.png[Create new project, width=300]
+
+Select the project of choice and in the new screen click on btn:[Add Server].
+
+In the following screen, you can define the::
+--
+* *Server Location* (Chose one from the offered)
+* *Image* (we use for this deployment Ubuntu)
+* *Type* (select any server type that matches your requirements)
+* *Networking* (we recommend using additionally IPv4)
```

Comment:
```suggestion
* *Networking* (we recommend using IPv4 as well as IPv6)
```

I am guessing a bit here, I suppose that "modern" Hetzner nowadays lets you have just an IPv6 networked server, and that you have to select IPv4 as an additional option.